src/item.h

```diff
@@ -54,6 +54,11 @@ struct tripoint;
 template<typename T>
 class ret_val;
 
+namespace enchant_vals
+{
+enum class mod : int;
+} // enchant_vals
```

Comment:
This needs to be `// namespace enchant_vals`

Reply:
Ah, I should have checked, not just gone off of memory. I'll tackle that in #41247